### ISSUE TITLE
front not running: certs volume unavailable

### ISSUE THREAD

**Issue by ook** [Mailu/helm-charts#6]
Installing from scratch master (13ff9eb8cab4ab78d9d77cf63ee518b8fa6144ae), secret `certificate-mailu` is still not created, so volume `certs` can't be mounted, and pod `front` can't start.

Note: I have to set `certManagerPre011: true`, since I have an old certmanager on that cluster.

**Comment by micw** [comment]
Do you have a working default clusterissuer?

**Comment by micw** [comment]
Please try to debug your cert manager (e.g. kubectl describe certificate/order/...).
If you run certmanager <=0.8, you won't get any certs (see https://community.letsencrypt.org/t/blocking-old-cert-manager-versions/98753).

**Comment by ook** [comment]
Indeed: my cert-manager `ClusterIssuer` was misconfigured for ACME resolver (CRD for Certificate was missing because I forgot to set ``. Thank you
Now I have a problem with nginx stealing mailu admin: I'll search on my side then eventually open another ticket for help.

**Comment by reavessm** [comment]
Hate to necro this post, but is there any way to get this running with out cert-manager?  I'm assuming setting TLS_FLAVOR of notls would skip the CertManager requirement, but I'm still getting the above issue.  Also, FWIW, I am running this on openshift 4.6 and will plan on using TLS on the routes (with edge termination), so I don't really want to bother with a cert-manager, especially for this POC
